app/models/vha_facility_adapter.rb

```diff
@@ -135,8 +135,10 @@ def service_whitelist
   ).freeze
 
   def self.mh_clinic_phone(attrs)
-    return '' if (attrs['MHClinicPhone']).blank?
-    result = attrs['MHClinicPhone'].to_s
+    val = attrs['MHClinicPhone']
+    val = attrs['MHPhone'] if val.blank?
+    return '' if val.blank?
+    result = val.to_s
```

Comment:
Historically I have no faith in the consistency of these values - I have seen null and '' in the JSON. 

There will be an opportunity to clean up some of the cruft in this logic and specs, and re-run the VCR cassettes once they deploy this schema change.